src/main/java/com/icssociety/automatedrequests/Header.java

```diff
@@ -0,0 +1,9 @@
+package com.icssociety.automatedrequests;
+
+import org.javalite.activejdbc.Model;
+import org.javalite.activejdbc.annotations.Table;
+
+@Table("request_headers")
+public class Header extends Model { // the Request model that is stored in the "requests" table
```

Comment:
Change this to RequestHeader, we will also be having a ResponseHeader so we need to differentiate between the two. Make sure to also change the comment.